3041b45

commit 3041b45
Author: Athanatos123 <alain.delree@gmail.com>
Date:   Thu Aug 13 22:38:39 2026 +0200

    feat: hide console window on Windows launch (start_silent.vbs)

diff --git a/installer-exe/alchess_setup.nsi b/installer-exe/alchess_setup.nsi
index 5d9c6eb..041766b 100644
--- a/installer-exe/alchess_setup.nsi
+++ b/installer-exe/alchess_setup.nsi
@@ -1942,8 +1942,8 @@ Section "Raccourci bureau" SecShortcut
     ; Reinitialiser le flag d'erreur avant CreateShortcut pour que IfErrors
     ; ne remonte pas une erreur laissee par une instruction anterieure.
     ClearErrors
-    CreateShortcut "$DESKTOP\AlChess.lnk" "$INSTDIR\2-Lancer_AlChess.bat" \
-        "" "$INSTDIR\alchess.ico" 0 SW_SHOWNORMAL "" "Launch AlChess"
+    CreateShortcut "$DESKTOP\AlChess.lnk" "wscript.exe" \
+        '"$INSTDIR\start_silent.vbs"' "$INSTDIR\alchess.ico" 0 SW_SHOWNORMAL "" "Launch AlChess"
     IfErrors shortcut_failed shortcut_ok
 
     shortcut_failed:
diff --git a/make_release.sh b/make_release.sh
index cc026a6..d9b270e 100755
--- a/make_release.sh
+++ b/make_release.sh
@@ -57,6 +57,7 @@ INCLUDE=(
   bootstrap_linux.sh
   1-Installer.bat
   2-Lancer_AlChess.bat
+  start_silent.vbs
   install_alchess.ps1
   start_alchess.ps1
   99-chessnutair.rules.example
diff --git a/start_silent.vbs b/start_silent.vbs
new file mode 100644
index 0000000..3435756
--- /dev/null
+++ b/start_silent.vbs
@@ -0,0 +1,13 @@
+' Lance 2-Lancer_AlChess.bat sans afficher de fenetre console.
+' Le log applicatif va deja dans alchess_log.txt, la console est inutile
+' en usage normal.
+
+Dim objShell, objFSO, strFolder, strBat
+
+Set objShell = CreateObject("WScript.Shell")
+Set objFSO = CreateObject("Scripting.FileSystemObject")
+
+strFolder = objFSO.GetParentFolderName(WScript.ScriptFullName)
+strBat = strFolder & "\2-Lancer_AlChess.bat"
+
+objShell.Run """" & strBat & """", 0, False
